source/gent/tier1_hortense.rst

```diff
@@ -60,33 +52,103 @@ Shared infrastructure:
 Getting access
 --------------
 
+.. _hortense_access_policy:
+
+Access policy
+*************
+
 **The Hortense VSC Tier-1 cluster can only be used by people with an active Tier-1 compute project.**
 
 See https://www.vscentrum.be/compute for more information on requesting access.
 
-To access the Tier-1 Hortense cluster, use SSH to connect to the dedicated login node with your VSC account:
+.. _hortense_login_nodes:
+
+Login nodes (SSH)
+*****************
+
+You can use SSH to connect to the login nodes of the Tier-1 Hortense cluster with your VSC account:
 
 * from the public internet, use ``tier1.hpc.ugent.be``
 * from within the VSC network, use ``tier1.gent.vsc``
 
 More general information about SSH login is available at :ref:`acccess_data_transfer`.
 
+There are 2 login nodes for Hortense: ``login55`` and ``login56``.
+When logging in using SSH, you will be (consistently) assigned to either of these login nodes,
+based on the IP address of the host you are connecting from.
+
+If you need to access a *specific* login node (for example because you have a ``screen`` or ``tmux`` session
+running there), just use ``ssh login56`` if you were assigned to ``login55`` (or vice versa).
```

Comment:
Reworded a bit more to:

```
If you need to access a *specific* login node (for example because you have a ``screen`` or ``tmux`` session
running there), just run "``ssh login56``" to jump to ``login56`` if you were logged in to ``login55``,
or use "``ssh login55``" to jump to ``login55`` from ``login56``.
```